### PR TITLE
Add Command line IDL Parser Tool

### PR DESCRIPTION
**Pull request by lauramg15**

### CODE REVIEW

File: tools/idl_validator.cpp [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#include <xtypes/xtypes.hpp>
+#include <xtypes/idl/idl.hpp>
+
+using namespace eprosima::xtypes;
+
+int main(
+    int argc, 
+    char** argv)
+{
+    if (argc == 1)
```
Comment (jamoralp) [inline tools/idl_validator.cpp]:
```suggestion
    if (argc != 2)
```

File: tools/idl_validator.cpp [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    {
+        std::cout << "The IDL must be passed as argument" << std::endl;
+        return -1;
+    }
+    else if (argc == 2)
```
Comment (jamoralp) [inline tools/idl_validator.cpp]:
```suggestion
    else
```

**Review by jamoralp — CHANGES_REQUESTED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/xtypes-manual-linux/158/)

**Review by jamoralp — APPROVED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/xtypes-manual-linux/159/)

**Pull request merged**